src/ride_type.cpp

```diff
@@ -226,6 +226,23 @@ const RideType *RideInstance::GetRideType() const
  * Immediately remove this ride from all voxels it currently occupies.
  */
 
+/**
+ * \fn void RideInstance::InsertIntoWorld()
+ * Ensure that this shop is linked into the voxels it is meant to occupy.
+ */
+
+/**
+ * \fn void ShouldDrawPiece::InsertIntoWorld()
```

Comment:
Class and function name are the wrong way around, but since you write this just above the function, you don't need this line at all, doxygen understands the text belongs to the next function by default